client/src/components/productTitle.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ProductTitle = ({ product }) => (
+  <div className="product-title">{ product.name }</div>
```

Comment:
unless you plan on adding more functionality to product title it would cut down on file size/lines of code to just write this one div as an element in the parent component

Reply:
That makes a lot of sense